diff --git a/out/codeSign/windowsSignAzureManager.js b/out/codeSign/windowsSignAzureManager.js
index 109dcc7f5ae4ffd766cdfc6faf8a3cdce23c639d..64ad3e54b77b3a0944b90059564c37835463f5e7 100644
--- a/out/codeSign/windowsSignAzureManager.js
+++ b/out/codeSign/windowsSignAzureManager.js
@@ -99,7 +99,7 @@ class WindowsSignAzureManager {
             TimestampRfc3161: timestampRfc3161 || "http://timestamp.acs.microsoft.com",
             TimestampDigest: timestampDigest || "SHA256",
             FileDigest: fileDigest || "SHA256",
-            Files: `"${options.path}"`,
+            Files: options.path,
         };
         const paramsString = Object.entries(params)
             .filter(([_, value]) => value != null)
