diff --git a/out/codeSign/windowsSignAzureManager.js b/out/codeSign/windowsSignAzureManager.js
index 109dcc7f5ae4ffd766cdfc6faf8a3cdce23c639d..057364882e4ae0469f437cd3bd93a78f208bab14 100644
--- a/out/codeSign/windowsSignAzureManager.js
+++ b/out/codeSign/windowsSignAzureManager.js
@@ -99,7 +99,7 @@ class WindowsSignAzureManager {
             TimestampRfc3161: timestampRfc3161 || "http://timestamp.acs.microsoft.com",
             TimestampDigest: timestampDigest || "SHA256",
             FileDigest: fileDigest || "SHA256",
-            Files: `"${options.path}"`,
+            Files: `"release\\${options.path.split('release')[1]}"`,
         };
         const paramsString = Object.entries(params)
             .filter(([_, value]) => value != null)
